release/assets/macos_cli_installation.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash -e
+
+USER_PROVIDED_PATH=$1
+
+main() {
+  cat << EOF
+   ______
+  / ____/___  ____    ( )_  _______
+ / /   / __ \/ __ \  / / / / / ___/
+/ /___/ /_/ / / / / / / /_/ / /
+\____/\____/_/ /_/_/ /\____/_/
+                /___/
+EOF
+
+  if [[ $USER_PROVIDED_PATH = "--help" ]]; then
+    print_help
+  fi
+  executable_path=${USER_PROVIDED_PATH:-'/usr/local/bin'}
```

Comment:
this looks quite odd. the `--help` is really a flag and seeing that you treat it as a provided path makes me think it is buggy. am i correct?

Reply:
no because it is caught in the above if statement which runs `print_help` and then exits the process